api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -236,7 +236,7 @@ message PlanState {
   // The status of the plan.
   enum Status {
     UNKNOWN = 0;
-    STARTING = 1;
+    STARTED = 1;
     EXECUTING = 2;
     COMPLETE = 3;
```

Comment:
I guess `COMPLETED` is a little better

https://english.stackexchange.com/questions/15400/complete-or-completed